assets/constants.ts

```diff
@@ -5740,6 +5740,7 @@ export default {
   // fields must be at most 1500 bytes, and UTF-8 encoded characters can be
   // up to 4 bytes long.
   "MAX_COMMIT_MESSAGE_LENGTH": 375,
+  "MAX_TEXT_MESSAGE_LENGTH": 10000,
```

Comment:
This seems too generic. Should the name indicate that the max length is for rejection message?

Reply:
The max length is not for the rejection message specifically. It is for the review message that the reviewer writes(which is optional if they accept the suggestion and compulsory if they reject it). This is why I thought that the constant name should be generic. What do you think?

Reply:
I think the updated "MAX_REVIEW_MESSAGE_LENGTH" is a better name. Thanks for updating!